src/index.ts

```diff
@@ -35,6 +35,11 @@ interface MyaccountOptions {
   authPage?: string;
 }
 
+const encodeConfig = (configs: object): string => {
```

Comment:
no need this function, use `qs.stringify(configs, { delimiter: ';' })`

Reply:
Oh! Thanks.